File: Sample/OBJ-C/Sample/Sample/Services/Service.m
//
//  Service.m
//  Sample
//
//  Created by Dabonot Francois on 03/01/2018.
//  Copyright © 2018 Dabonot Francois. All rights reserved.
//

#import "Service.h"
#import "Data.h"


@implementation Service

- (instancetype)initWithWindow:(UIWindow *)window
{
    if (self = [super initWithWindow:window])
    {
        _data = [Data new];
    }
    return (self);
}

@end

File: Sample/Swift/Sample/Sample/Services/Service.swift
//
//  Service.swift
//  Sample
//
//  Created by Dabonot Francois on 03/01/2018.
//  Copyright © 2018 Dabonot Francois. All rights reserved.
//

import Foundation

class Service: BaseServices {
    var data:IData!;
    
    override init(window:UIWindow) {
        super.init(window: window)
        data = Data();
    }
    
}
